src/main/java/frc/trigon/robot/misc/objectdetectioncamera/ObjectDetectionCamera.java

```diff
@@ -21,36 +26,70 @@ public void periodic() {
         Logger.processInputs(hostname, objectDetectionCameraInputs);
     }
 
+    /**
+     * Starts the tracking of the best visible target and remains tracking that target until it is no longer visible.
+     * Tracking an object is locking on to one target and allows you to stay locked on to one target.
+     * This is used when there is more than one target that might change as the robot moves to stabilize the result and ensure that it is following the same target that it started with.
+     * This needs to be called each time you want to track a new object.
+     */
```

Comment:
wrong, as you said before `and remains tracking that target until it is no longer visible`. 
explain what happens when no targets are visible